### PR TITLE
Made the notices look nicer :)

### PR DESCRIPTION
**Pull request by EvelynDRose**
hope it looks good 😎

### CODE REVIEW

File: app/assets/stylesheets/stylesMembers.css [hunk 1]
```diff
@@ -34,4 +34,39 @@
...
+
+.edit
+{
+    background-color:#fdff71;
+}
+
+.show
+{
+    background-color:#71ff71;
+}
```
Comment (JonathanKonderla) [inline app/assets/stylesheets/stylesMembers.css:72]: nice style B)

File: app/views/points/new.html.erb [hunk 2]
```diff
@@ -14,11 +14,10 @@
...
                 <% if(@current_user.isOfficer) %>
-                    <%= link_to "View All Points", points_path %>
-                    <%= link_to "View All Users", portal_view_members_path %>
-                    <%= link_to "Search Users", users_path%>
-                    <%= link_to "Announcement", new_announcement_path%>
-                    <%# <li> link_to "Edit Calender", --- </li> %> <!-- Fill this in later when calender is done :) -->
+                <%= link_to "View All Points", points_path %>
+                <%= link_to "View All Users", portal_view_members_path %>
+                <%= link_to "Search Users", users_path%>
+                <%= link_to "Announcement", new_announcement_path%>
```
Comment (JonathanKonderla) [inline app/views/points/new.html.erb]: lines should be indented

File: app/views/points/show.html.erb [hunk 3]
```diff
@@ -14,11 +14,10 @@
...
                 <% if(@current_user.isOfficer) %>
-                    <%= link_to "View All Points", points_path %>
-                    <%= link_to "View All Users", portal_view_members_path %>
-                    <%= link_to "Search Users", users_path%>
-                    <%= link_to "Announcement", new_announcement_path%>
-                    <%# <li> link_to "Edit Calender", --- </li> %> <!-- Fill this in later when calender is done :) -->
+                <%= link_to "View All Points", points_path %>
+                <%= link_to "View All Users", portal_view_members_path %>
+                <%= link_to "Search Users", users_path%>
+                <%= link_to "Announcement", new_announcement_path%>
```
Comment (JonathanKonderla) [inline app/views/points/show.html.erb]: lines should be indented

File: app/views/portal/home.html.erb [hunk 4]
```diff
@@ -10,17 +10,16 @@
...
             <% if(@current_user.isOfficer) %>
-                <%= link_to "View All Points", points_path %>
-                <%= link_to "View All Users", portal_view_members_path %>
-                <%= link_to "Search Users", users_path%>
-                <%= link_to "Announcement", new_announcement_path%>
-                <%# <li> link_to "Edit Calender", --- </li> %> <!-- Fill this in later when calender is done :) -->
+            <%= link_to "View All Points", points_path %>
+            <%= link_to "View All Users", portal_view_members_path %>
+            <%= link_to "Search Users", users_path%>
+            <%= link_to "Announcement", new_announcement_path%>
```
Comment (JonathanKonderla) [inline app/views/portal/home.html.erb]: should be indented

File: app/views/portal/view_members.html.erb [hunk 5]
```diff
@@ -10,13 +10,13 @@
...
             <% if(@current_user.isOfficer) %>
-                <%= link_to "View All Points", points_path %>
-                <%= link_to "View All Users", portal_view_members_path %>
-                <%= link_to "Search Users", users_path%>
-                <%= link_to "Announcement", new_announcement_path%>
-                <%# <li> link_to "Edit Calender", --- </li> %> <!-- Fill this in later when calender is done :) -->
+            <%= link_to "View All Points", points_path %>
+            <%= link_to "View All Users", portal_view_members_path %>
+            <%= link_to "Search Users", users_path%>
+            <%= link_to "Announcement", new_announcement_path%>
```
Comment (JonathanKonderla) [inline app/views/portal/view_members.html.erb]: should be indented

**Review by JonathanKonderla — CHANGES_REQUESTED** [review]

**Comment by armenaa652** [comment]
can confirm, looks good

**Pull request merged**